src/algebraic_geometry/structure_sheaf.lean

```diff
@@ -328,33 +340,265 @@ def stalk_iso_Type (x : prime_spectrum R) :
   ⟨structure_sheaf_stalk_to_fiber_injective R x,
    structure_sheaf_stalk_to_fiber_surjective R x⟩).to_iso
 
--- PROJECT: Improve this to an isomorphism of rings.
+
 /-
-/--
-TODO: this should follow easily from `forget CommRing` preserving filtered colimits.
--/
-def compare_stalks (x : prime_spectrum R) :
-  (forget CommRing).obj ((structure_sheaf R).presheaf.stalk x) ≅ (structure_sheaf_in_Type R).presheaf.stalk x :=
-sorry
 
-/--
-TODO:
+Mathematical Notations:
```

Comment:
Maybe `Notation in this comment`? Note that you can use LaTeX in comments.

Reply:
This is invisible in the auto-doc-gen anyway right

Reply:
Also I don't know what to do with my half-code half-mathematical-notation mess so I wouldn't want to put it in the auto-doc-gen

Reply:
I agree with Johan though that saying "Mathematical Notations" here is a bit confusing.